kinetic/distribution.yaml

```diff
@@ -9215,6 +9215,12 @@ repositories:
       url: https://github.com/amineHorseman/pioneer_teleop.git
       version: master
     status: maintained
+  pipeline_planner:
+    source:
```

Comment:
We can add this, but this does not add it to the documentation index.  If you want that, you need to add a `doc` section as well; see the `platform_automation_msgs` package just below for an example.